### ISSUE TITLE
Using custom trained detector with ByteTrack

### ISSUE THREAD

**Issue by MoaazAbdulrahman** [ifzhang/ByteTrack#115]
Thank you for this great work. I see that you are not using any re-id. In that case, can I use my custom trained detetor and pass the bounding boxes to Byte tracker without running any additional training on ByteTracker?

Thanks?

**Comment by pkrskr** [comment]
Yes @MoaazAbdulrahman the github home page explains this in the section titled "Applying BYTE to other trackers"

**Comment by thaihaquang** [comment]
You should read the section "Combining BYTE with other detectors"
